docs/reference/sql/language/indices.asciidoc

```diff
@@ -88,7 +88,7 @@ requires the keyword `LIKE` for SQL `LIKE` pattern.
 [[sql-index-frozen]]
 === Frozen Indices
 
-By default, {es-sql} doesn't search <<freeze-index-api,frozen indices>>. To
+By default, {es-sql} doesn't search frozen indices. To
```

Comment:
I'd link to the unfreeze index API so new users know this isn't related to the frozen data tier.
```suggestion
By default, {es-sql} doesn't search <<unfreeze-index-api,frozen indices>>. To
```